frontend/src/components/Restaurants/index.jsx

```diff
@@ -11,12 +15,15 @@ const propTypes = {
 const Restaurants = ({ restaurants, error }) => {
 
     const restaurantsDisplay = (restaurants) => {
-        return restaurants.map((restaurant, i) => (
-            <Fragment key = {i}>
-                <p><b>{restaurant.name}</b> </p>
-                <AddressDisplay address={restaurant.address} />
-            </Fragment>
-        ));
+        return restaurants.map((restaurant, i) => {
+            let restaurantLink = `http://${window.location.hostname}:3000${PATH_REVIEWS}`.replace(PATH_VARIABLE_RESTAURANT_ID, restaurant.id)
```

Comment:
Let's make a constant for the hostname / port #

Reply:
Done in [270a580](https://github.com/NickPriv/FryRank/pull/21/commits/270a5806712ecc2080dc3256cf5936d155029a8e).